clojure/README.md

```diff
@@ -0,0 +1,57 @@
+# clojure
+
+Clojure is a robust, practical, and fast programming language with a set of useful features that together form a simple, coherent, and powerful tool.
+
+## Maintainers
+
+* The Habitat Maintainers: <humans@habitat.sh>
+
+## Type of Package
+
+Binary package
+
+## Usage
```

Comment:
Usage here (per [the README template](https://github.com/habitat-sh/core-plans/blob/master/README_TEMPLATE_FOR_PLANS.md)) should guide the user as to how to use this plan, not an output of the command line help.

For example, some use cases are: How does the user install and use this plan to operate the binary? How would the user ideally go about using this plan as a dependency of their own plan or in a configuration plan (if applicable)?

Reply:
Updated, please review.